test.env.tmpl

```diff
@@ -15,4 +15,5 @@ SPACELIFT_PROVIDER_TEST_GITHUB_ENTERPRISE_APP_ID=
 
 SPACELIFT_PROVIDER_TEST_BITBUCKET_DATACENTER_API_HOST=
 SPACELIFT_PROVIDER_TEST_BITBUCKET_DATACENTER_WEBHOOK_SECRET=
+SPACELIFT_PROVIDER_TEST_BITBUCKET_DATACENTER_WEBHOOK_URL=
```

Comment:
Not sure about what should be put in `test.yml` and `test-prod.yml`

Reply:
I have an idea how to f

Reply:
I have an idea how to figure it out